django/db/backends/__init__.py

```diff
@@ -463,6 +463,11 @@ def close_if_unusable_or_obsolete(self):
         Closes the current connection if unrecoverable errors have occurred,
         or if it outlived its maximum age.
         """
+
+        """ Returning from here as db connections are getting close on each request, 
+        causing overhead on server to make new connections to db"""
+        return
```

Comment:
Need to explore https://docs.djangoproject.com/en/3.2/ref/settings/#conn-max-age
Also available in django 1.6.11 and is being used here as `self.close_at`